targets/gtk-extra

```diff
@@ -8,12 +8,16 @@ DESCRIPTION='GTK-based tools including gdebi, gksu, and a simple browser.'
 
 ### Append to prepare.sh:
 install_dummy network-manager network-manager-gnome
-install gdebi gksu netsurf-gtk
+install gdebi gksu
 
-# Add netsurf-gtk to Debian-alternatives if not already there
-netsurf='/usr/bin/netsurf-gtk'
-for link in x-www-browser gnome-www-browser; do
-    if ! update-alternatives --query "$link" | grep -q "$netsurf"; then
-        update-alternatives --install "/usr/bin/$link" "$link" "$netsurf" 10
+for BROWSER in netsurf-gtk dillo; do
```

Comment:
Very nicely structured.  If no browsers succeed, can you fail out?  One easy way is like this:

```
for BROWSER in netsurf-gtk dillo ""; do
    test -n "$BROWSER"
    if install .....
```

Then if dillo doesn't succeed, it'll try the blank string which will fail the test.